src/applications/edu-benefits/utils/labels.jsx

```diff
@@ -193,13 +193,15 @@ export const directDepositWarning = (
           <b>or</b>
         </li>
         <li>
-          Call <a href="tel:8003331795">800-333-1795</a>
+          Call{' '}
+          <va-telephone contact="8003331795"></va-telephone>
```

Comment:
Nice use of <va-telephone>, but you can make these self closing (`<va-telephone contact="8003331795" />`).

Reply:
resolved